server/src/Stories/Stories.controller.ts

```diff
@@ -33,26 +33,40 @@ export const getAllStoriesByProject = async (req: Request, res: Response) => {
   }
 };
 
-/**
- * @body projectName: string | number 대상 프로젝트의 id값
- * @body name: string 새롭게 생성하는 에픽의 이름
- * @response id: number 새롭게 생성된 에픽의 id값
- */
+export const getStoryById = async (req: Request, res: Response) => {
+  try {
+    if (!req.params.id) {
+      throw new Error('유효한 ID 가 존재하지 않습니다.');
+    }
+    const { id } = req.params;
+    const result = await getRepository(Stories).findOne(id);
+    if (!result) {
+      throw new Error(`해당 ${id} 를 조회할 수 없습니다`);
+    }
+    res.status(200).json(result);
```

Comment:
express res.json의 디폴트 값이 status 코드 200인 것으로 알고 있습니다.
```suggestion
    res.json(result);
```